types/src/front/lib/assistant.ts

```diff
@@ -144,7 +144,7 @@ export const GEMINI_PRO_DEFAULT_MODEL_CONFIG = {
   providerId: "google_ai_studio" as const,
   modelId: GEMINI_1_5_PRO_LATEST_MODEL_ID,
   displayName: "Gemini Pro 1.5",
-  contextSize: 1_000_000,
+  contextSize: 400_000,
```

Comment:
Can you add a comm to explain the POST limit here?

Reply:
And edit line 151 (description field) if we go with that 🙏🏻